INSTALL.md

```diff
@@ -47,6 +47,8 @@ $ cmake .. -DOpenCV_DIR=${OPENCV_INSTALL}/share/share/OpenCV/
 $ make -j `nproc`
 ```
 
+For Windows you will need to add `-DCCTAG_WITH_CUDA:BOOL=OFF` as CUDA support is not yet working on Windows. You will most likely also need to generate build files for NMAKE or Visual Studio, and build with those tools.
```

Comment:
I suggest:
* few line above, explicitly say the version of eigen:
```md
 - Eigen3 (libeigen3-dev) >= 3.3.4  (NOTE: in order to have Cuda support on Windows, at least version 3.3.9 is required)
``` 
* The here I suggest just to explain the use of `CCTAG_WITH_CUDA`

```suggestion
Use `-DCCTAG_WITH_CUDA:BOOL=ON` to enable (default) or disable the cuda support. On Windows, it must be set to `OFF` if the Eigen version is < 3.3.9
```